packages/react/src/components/OverflowMenu/OverflowMenu.js

```diff
@@ -525,26 +526,29 @@ class OverflowMenu extends Component {
     };
 
     return (
-      <ClickListener onClickOutside={this.handleClickOutside}>
-        <div
-          {...other}
-          role="menu"
-          aria-haspopup
-          aria-expanded={this.state.open}
-          className={overflowMenuClasses}
-          onKeyDown={this.handleKeyPress}
-          onBlur={this.handleBlur}
-          onClick={this.handleClick}
-          aria-label={ariaLabel}
-          id={id}
-          tabIndex={tabIndex}
-          ref={mergeRefs(ref, this.bindMenuEl)}>
-          <IconElement {...iconProps}>
-            {iconDescription && <title>{iconDescription}</title>}
-          </IconElement>
-          {open && wrappedMenuBody}
-        </div>
-      </ClickListener>
+      <React.Fragment>
+        <ClickListener onClickOutside={this.handleClickOutside}>
+          <div
+            {...other}
+            role="button"
+            aria-haspopup
+            aria-expanded={this.state.open}
+            className={overflowMenuClasses}
+            onKeyDown={this.handleKeyPress}
+            onBlur={this.handleBlur}
+            onClick={this.handleClick}
+            aria-label="Menu"
```

Comment:
Instead of killing `ariaLabel`, do you want to change `defaultProps` to make `Menu` the default?